test/functional/apps/management/_index_pattern_create_delete.js

```diff
@@ -28,12 +28,11 @@ export default function ({ getService, getPageObjects }) {
           .then(id => indexPatternId = id);
       });
 
-      it('should have index pattern in page header', function () {
-        return PageObjects.settings.getIndexPageHeading().getVisibleText()
-        .then(function (patternName) {
-          screenshots.take('Settings-indices-new-index-pattern');
-          expect(patternName).to.be('logstash-*');
-        });
+      it('should have index pattern in page header', async function () {
+        const indexPageHeading = await PageObjects.settings.getIndexPageHeading();
+        const patternName = await indexPageHeading.getVisibleText();
```

Comment:
I'm not super well versed in async/await, but I thought it was basically just like wrapping the call in a then handler. If I'm right about that, couldn't this be written like so?

```js
const patternName = await PageObjects.settings.getIndexPageHeading().getVisibleText();
```